Gemfile

```diff
@@ -44,8 +47,8 @@ gem 'oink'
 gem 'pry'
 
 gem 'iiif-image-api', git: 'https://github.com/samvera-labs/iiif-image-api.git', branch: 'main'
-gem 'iiif-presentation', git: 'https://github.com/benwbrum/osullivan', branch: 'service_is_array'
-gem 'riiif', git: 'https://github.com/BrumfieldLabs/riiif.git', branch: 'quote-paths-for-shell'
+gem 'iiif-presentation'
+gem 'riiif'
```

Comment:
Checked these forks and compared with main, it seems the changes applied here in forks are done already in main (merged from another PR I guess).

So we should be good to get these from main repo.